.circleci/config.yml

```diff
@@ -8,8 +8,22 @@ jobs:
       - checkout
       - run: sudo chown -R 999 e2e
       - run: sh run-e2e.sh
+  cypress:
+    machine:
+      image: ubuntu-1604:202004-01
```

Comment:
Do they have more recent image? Ubuntu 16.04 will be supported until April 2021 so it's less than a year.

Reply:
By then, we might move this to github actions, but I am not sure.

Reply:
Indeed the plan is to move to github actions